test/integration/Sandwich.t.sol

```diff
@@ -0,0 +1,221 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.22;
+
+import { console2 as console } from "forge-std/console2.sol";
+import { Lib } from "hyperdrive/test/utils/Lib.sol";
+import { HyperdriveUtils } from "hyperdrive/test/utils/HyperdriveUtils.sol";
+import { EverlongTest } from "../harnesses/EverlongTest.sol";
+
+contract Sandwich is EverlongTest {
+    using Lib for *;
+    using HyperdriveUtils for *;
+
+    function setUp() public override {
+        super.setUp();
+        deployEverlong();
+    }
+
+    /// @dev Tests the short sandwich scenario with no idle liquidity.
+    function testFuzz_sandwich_short_instant_no_idle(
+        uint256 _shortAmount,
+        uint256 _attackerDepositAmount,
+        uint256 _bystanderDepositAmount
+    ) external {
+        TARGET_IDLE_LIQUIDITY_PERCENTAGE = 0;
+        MAX_IDLE_LIQUIDITY_PERCENTAGE = 0;
+        super.setUp();
```

Comment:
Why do we need to call `super.setUp()` in each of these functions? If it has to do with calling `deployEverlong` in setup and subsequently in each of these tests, why don't we just remove the whole `setUp` function, and then it will automatically call `super.setUp`?

Reply:
We don't, will remove